public/video-ui/src/reducers/editStateReducer.js

```diff
@@ -1,10 +1,7 @@
-export default function video(state = {
-    metadataEditable: false,
-    youtubeEditable: false
-  }, action) {
+export default function video(state = false, action) {
```

Comment:
Because we used to have two forms to track so they were in an object but now we just have to track the edit state of one form so it might as well be a bool.